### PR TITLE
[ready] average aoi completeness

### PR DESCRIPTION
**Pull request by geohacker**
cc @kamicut @ascalamogna 

This has a tile-reduce worker that reads each AoI and computes average completeness and adds an attribute `averageCompleteness` in building-stats.json. Not entirely sure if this approach, we'll have to see how it fits to the visualisation.

Do we also want to store population average? I'm not sure what the units are. I think it's number of people per tile.

### CODE REVIEW

File: workers/completeness-aoi/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        ]
+    })
+    .on('reduce', function(data) {
+        if (data.sumIndex) {
+            units = units + data.units;
+            averageCompleteness = averageCompleteness + data.sumIndex;
+        }
+    })
+    .on('end', function() {
+        averageCompleteness = averageCompleteness/units;
```
Comment (kamicut) [inline workers/completeness-aoi/index.js]: Is this possible to be a divide by zero error? We should do a quick check for it.

**Review by kamicut — CHANGES_REQUESTED** [review]
Just needs a quick check.

File: workers/completeness-aoi/index.js (hunk 1 above)
Comment (geohacker) [reply workers/completeness-aoi/index.js]: It shouldn't if the predictions are correct, but I'll add a check anyway. Thanks for catching!

**Comment by geohacker** [comment]
fixes https://github.com/hotosm/osma-health/issues/10

**Pull request merged**